release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -891,6 +891,19 @@ With this release, replacement control plane nodes are assigned to the correct i
 [discrete]
 [id="ocp-4-13-management-console-bug-fixes"]
 ==== Management Console
+* Previously, user permissions were not checked when rendering the *Create Pod* button, and the button rendered for users without needed permissions. With this update, user permissions are checked when rendering the *Create Pod* button, and it renders for users for users with the needed permissions. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2005232[*BZ#2005232*])
+
+* Previously, the `Pod` resource had a `PDB` *_add_*, *_edit_*, and *_remove_* actions in the Pod resource action menu that are not required. With this update, the actions are removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110565[*BZ#2110565*])
+
+* Previously, the `PodDisruptiojnBudget` field on the *Details* page had an incorrect help message. With this update, the help message is now more descriptive. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084452[*BZ#2084452*])
+
+* Previously, when navigating to the root path of the console, the URL redirects to the *Overview* page even if metrics are disabled and it doesn't appear in the navigation menu. With this update, when clicking the masthead logo or navigating to the root path of the console, the URL will redirect to the *project list* page if metrics are disabled. (link:https://issues.redhat.com/browse/OCPBUGS-3033[*OCPBUGS-3033*])
+
+* Previously, the cluster dropdown was positioned so it is not always visible making it unclear which cluster you are viewing. With this update, the cluster dropdown is now in the masthead so the cluster dropdown is always visible, and you can always see which cluster you are viewing. (link:https://issues.redhat.com/browse/OCPBUGS-7089[*OCPBUGS-7089*])
```

Comment:
```suggestion
* Previously, the cluster dropdown was positioned so that it was not always visible, making it unclear which cluster you were viewing. With this update, the cluster dropdown is now in the masthead so the cluster dropdown is always visible, and you can always see which cluster you are viewing. (link:https://issues.redhat.com/browse/OCPBUGS-7089[*OCPBUGS-7089*])
```